ProblemsSolution.Tests/ProblemsSolutionTest.cs

```diff
@@ -0,0 +1,32 @@
+using NUnit.Framework;
+
+namespace ProblemsSolution.Tests
+{
+    [TestFixture]
+    public class Tests
+    {
+
+        [TestCase (new int[] {1,2,3,4,5}, ExpectedResult=6, Description="Check for array with even numbers.")]
+        [TestCase (new int[] {3, 7, 0, 1}, ExpectedResult=0, Description="Check for array with even numbers and a zero.")]
+        public int SumArrayEvenNumbersTest(int[] numbers)
+        {
+            return ProblemsSolution.ArrayProblemsSolution.SumArrayEvenNumbers(numbers);
+        }
+
+        [TestCase (new int[] {12, 22, 1, 3}, ExpectedResult=new int[] {12, 22}, Description="Check for array with more than 2 elements.")]
+        [TestCase (new int[] {3}, ExpectedResult=new int[] {3}, Description="Check for single element array.")]
+        [TestCase (new int[] {7, 3}, ExpectedResult=new int[] {3, 7}, Description="Check for 2 element array.")]
+        [TestCase (new int[] {}, ExpectedResult=new int[] {}, Description="Check for empty array.")]
+        public int[] BigestTwoNumbersArrayTest(int[] numbers)
+        {
+            return ProblemsSolution.ArrayProblemsSolution.BigestTwoNumbersArray(numbers);
+        }
```

Comment:
Style looks consistent except this missing empty new line, Wonder what happened here?

Reply:
I didn't notice it.

Reply:
Fixed af59a0081a4531bc447a1c0ef6d2af8b1b5da9f9.